<?xml version="1.0" encoding="UTF-8"?>

<!--
  Copyright (c) 2021 by Jannik Alber.
  All rights reserved.
  -->

<ui version="4.0" connectslotsbyname="false">
	<class>Workbench</class>
	
	<widget class="QMainWindow" name="Workbench">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>920</width>
				<height>640</height>
			</rect>
		</property>
		
		<property name="styleSheet">
			<!-- This applies only for inactive views. The active view's color will be set back to the default color. -->
			<string notr="true">
				QTabWidget#nova-content-tab-view QTabBar::tab {
					width: 175px;
					color: dimgray;
				}
			</string>
		</property>
		
		<widget class="QStackedWidget" name="stwCentralWidget">
			<property name="currentIndex">
				<number>0</number>
			</property>
			
			<widget class="QLabel" name="lblEmptyView">
				<property name="alignment">
					<enum>Qt::AlignCenter</enum>
				</property>
				
				<property name="textFormat">
					<enum>Qt::MarkdownText</enum>
				</property>
				
				<property name="frameShape">
					<enum>QFrame::Box</enum>
				</property>
				
				<property name="frameShadow">
					<enum>QFrame::Sunken</enum>
				</property>
				
				<property name="styleSheet">
					<string notr="true">background-color: palette(base);</string>
				</property>
			</widget>
		</widget>
		
		<widget class="QMenuBar" />
		
		<widget class="QStatusBar">
			<property name="maximumHeight">
				<number>21</number>
			</property>
			
			<widget class="QLabel" name="lblConnectionState" />
			
			<widget class="QWidget" name="wdgProgress">
				<layout class="QHBoxLayout">
					<item>
						<widget class="QLabel" name="lblProgressDescription" />
					</item>
					
					<item>
						<widget class="QProgressBar" name="prbProgress">
							<property name="maximumHeight">
								<number>15</number>
							</property>
							
							<property name="visible">
								<bool>false</bool>
							</property>
							
							<property name="alignment">
								<enum>Qt::AlignCenter</enum>
							</property>
						</widget>
					</item>
				</layout>
			</widget>
			
			<widget class="QWidget" name="wdgNotificationBar">
				<property name="visible">
					<bool>false</bool>
				</property>
				
				<layout class="QHBoxLayout">
					<item>
						<widget class="QLabel" name="lblNotificationIcon">
							<property name="sizePolicy">
								<sizepolicy hsizetype="Maximum" vsizetype="Preferred" />
							</property>
						</widget>
					</item>
					
					<item>
						<widget class="QLabel" name="lblNotification">
							<property name="sizePolicy">
								<sizepolicy hsizetype="Maximum" vsizetype="Preferred" />
							</property>
							
							<property name="textInteractionFlags">
								<enum>Qt::TextSelectableByMouse</enum>
							</property>
						</widget>
					</item>
					
					<item>
						<widget class="QLabel" name="lblNotificationLinks" />
					</item>
				</layout>
			</widget>
		</widget>
	</widget>
</ui>
